book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
```

Comment:
- [ ] You are adding a rentals to a rentals array that does not exist. Please initialize the rentals array here so that whatever rental you add to a book can be added to this array. For example
`@rentals = []` 👍